nemo/collections/nlp/models/machine_translation/mt_enc_dec_config.py

```diff
@@ -99,25 +101,25 @@ class AAYNBaseConfig(MTEncDecModelConfig):
         tokens_in_batch=512,
         clean=True,
         shuffle=True,
-        cache_ids=True,
-        use_cache=True,
+        cache_ids=False,
```

Comment:
not quite sure what's the interaction with tarred datasets and `cache_ids` and `use_cache` parameter

Reply:
I agree, but not in this PR so we give some time for users to adjust. I'll add some deprecation warnings though.